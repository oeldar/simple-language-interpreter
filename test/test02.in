print(1 = 1);
print(1 <> 1);
print(1 < 5);
print(1 > 5);

print(1 >= 3);
print(3 <= 3);

print(2 + 5);
print(2 - 5);
print(10 / 5);
print(2 * 5);

print(--10);
