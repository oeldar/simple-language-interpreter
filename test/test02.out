1
0
1
0
0
1
7
-3
2
10
10
